### DISCUSSION TITLE
Baffled Newbie in need of help

### DISCUSSION THREAD

**Discussion by OliverTint** [ImageMagick/ImageMagick#3611] [Help]
Hello everyone I'm struggling to find out how to use ImageMagick for just one purpose, I want to measure the actual speed of an animated gif. I've downloaded ImageMagick version 7.0.11-Q16-HDRI onto my pc which has the Windows 10 operating system.

No where within the ImageMagick folders can I find an identify or -identify executable and when I do a cmd command entering either of these commands I get an error message saying they are neither an internally nor an externally available function.

I can find an executable, identify.html, and when I run it it opens in Dreamweaver but try as I might I can find a way to process an animated gif in it.

Have I downloaded the wrong version of ImageMagick or do the identify and/or -identify cmd functions have to be downloaded separately?

Apologies in advance if this question has been dealt with previously and if it has can someone point me to the subject please?

Kind regards,

Oliver

**Comment by snibgo** [comment]
"identify.html" is not an executable. From the extension, "html", it is a web page.

Where did you download ImageMagick from? (What web site?) Did you install it on your computer? What directory (aka "folder") did you install it in? What files are in that directory?
